Makefile

```diff
@@ -0,0 +1,55 @@
+# collision prediction pipe
+# author: MDS-2021-22 block3 group21
+# date: 2021-11-30
+
+# all
+all : data/processed/train.csv \
```

Comment:
The makefile ran without errors? I haven't seen this format where each target is separated by a "\" so I want to ensure.